src/picongpu/include/plugins/ResourceLog.hpp

```diff
@@ -0,0 +1,244 @@
+/**
+ * Copyright 2016 Erik Zenker
+ *
+ * This file is part of libPMacc.
+ *
+ * libPMacc is free software: you can redistribute it and/or modify
+ * it under the terms of either the GNU General Public License or
+ * the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * libPMacc is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License and the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * and the GNU Lesser General Public License along with libPMacc.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// PMacc
+#include "Environment.hpp"
+#include "mappings/simulation/ResourceMonitor.hpp"
+
+// PIConGPU
+#include "plugins/ILightweightPlugin.hpp"
+#include "ILightweightPlugin.hpp"
+#include "simulation_defines.hpp"
+
+// Boost
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
```

Comment:
good question.
I think no, since I can not find [it](http://www.boost.org/doc/libs/1_60_0/doc/html/property_tree.html) in the list of [to-be-compiled boost libraries](https://gist.github.com/ax3l/7450482).